web/js/containers/sidebar/sidebar.js

```diff
@@ -192,19 +192,53 @@ class Sidebar extends React.Component {
       isDistractionFreeModeActive,
       isEmbedModeActive,
       selectedDate,
+      isMobile,
     } = this.props;
     const permalink = getPermalink(history.location.search, selectedDate);
     const WVLogoTitle = isEmbedModeActive
       ? 'Click to Open This @NAME@ Map in a New Tab'
       : 'Click to Reset @NAME@ to Defaults';
     const embedWVLogoLink = isEmbedModeActive ? permalink : '/';
+    const mobileImgURL = '../../../brand/images/wv-logo-mobile.svg?v=@BUILD_NONCE@';
+    const desktopImgURL = '../../../brand/images/wv-logo.svg?v=@BUILD_NONCE@';
+
+    const mobileStyle = {
+      background: `url(${mobileImgURL}) no-repeat center rgb(40 40 40 / 85%)`,
+      display: 'block',
+      height: '42px',
+      width: '56px',
+      padding: '5px',
+      top: '10px',
+      left: '10px',
+      borderRadius: '5px',
+      border: '1px solid #333',
+      position: 'absolute',
+
+    };
+
+    const desktopStyle = {
+      background: `url(${desktopImgURL}) no-repeat center rgb(40 40 40 / 85%)`,
+      display: 'block',
+      width: '286px',
+      height: '55px',
+      padding: '5px 0',
+      position: 'absolute',
+      top: '10px',
+      left: '10px',
+      border: '1px solid #333',
```

Comment:
I guess we want to re-write this a bit to not throw this error:
![Screen Shot 2022-09-13 at 9 02 30 AM](https://user-images.githubusercontent.com/1480000/189937221-d78969ce-6008-43c2-b16b-d16cc76f3bbe.png)

Reply:
Broke out the borderBottom styling to remove this warning message.